draft-ietf-rtcweb-jsep.xml

```diff
@@ -4076,6 +4076,65 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
             process, as a remote offer or answer, unmodified SDP coming
             from any other implementation of this specification.</t>
         </section>
+        <section title="Processing RTP packets"
+                 anchor="sec.rtp.demux">
+          <t>When an RTP packet is received by a transport (typically a
+          DtlsTransport), that packet MUST be routed to the correct
+          RtpReceiver. For each transport, the following steps MUST be
```

Comment:
I also believe that we need to describe how table entries are brought into existence or removed (and what methods are involved).  Since the table entries are per DtlsTransport, they cannot exist prior to construction of the DtlsTransport (e.g. when receiver.transport is null).  

IMHO, for an Offerer, table entries are set/removed by calling setRemoteDescription, and for an Answerer, by calling setLocalDescription.  

Since transceiver.setDirection and transceiver.stop set the negotiation-needed flag, I do not believe that these methods result in the immediate removal of table entries  (e.g. calling transceiver.setDirection(direction) with a direction value of "sendonly" or "inactive" does not remove table entries until negotiation completes).